manager/allocator/network.go

```diff
@@ -948,15 +975,25 @@ func (a *Allocator) procUnallocatedServices(ctx context.Context) {
 	}
 }
 
-func (a *Allocator) procUnallocatedTasksNetwork(ctx context.Context) {
+func (a *Allocator) procTasksNetwork(ctx context.Context, onRetry bool) {
 	nc := a.netCtx
-	allocatedTasks := make([]*api.Task, 0, len(nc.unallocatedTasks))
+	quiet := false
+	toAllocate := nc.pendingTasks
+	allocatedTasks := make([]*api.Task, 0, len(toAllocate))
```

Comment:
This line should go below the if block, after which we know what `toAllocate` points to